frontend/src/locales/de.json

```
@@ -228,7 +228,11 @@
     },
     "form": {
       "base": {
+        "eColorField": {
+          "parseError": "Bitte valide Farbe eingeben."
```

Depends on the taste:
```suggestion
          "parseError": "Bitte gültige Farbe eingeben."
```